bluemira/builders/coil_supports.py

```diff
@@ -283,7 +284,13 @@ def build_xyz(
         # Finally, make the floor block
         shape_list.append(self._make_floor_block(floatify(v1.x), floatify(v4.x)))
         shape = boolean_fuse(shape_list)
-        component = PhysicalComponent("ITER-like gravity support", shape)
+        component = PhysicalComponent(
+            "ITER-like gravity support",
+            shape,
+            material=get_cached_material(
+                self.build_config.get("material", {}).get("GS")
```

Comment:
The component material maybe should go as a class var so it can be accessed outside? Same as we do in some places for the PhysicalComponent name (and you've done below).